src/scheduler/get_4byte.c

```diff
@@ -113,35 +99,29 @@ get_sched_cmd(int sock, int *val, char **jid)
 /**
  *
  * @brief
- * 		This is non-blocking version of get_sched_cmd().
+ * 		This is non-blocking version of get_sched_cmd()
+ *
+ * @param[in]	fd	-	connection to server
+ * @param[in/out]	cmd	-	sched command received from server
  *
- * @param[in]	sock	-	communication endpoint to the server`
- * @param[in]	val	-	points to the value of the command sent by server.
- * @param[in]   jid	-	if *val is SCH_SCHEDULE_AJOB, then return jobid in
- *						*jid.
  * @return	int
  * @retval	0	for EOF,
  * @retval	+1	for success
  * @retval	-1	for error.
- *
- * @note
- *		Callers need to free the value pointed to by 'jid' after returning
- *		from this call.
  */
 
 int
-get_sched_cmd_noblk(int sock, int *val, char **jid)
+get_sched_cmd_noblk(int fd, sched_cmd_t *cmd)
```

Comment:
Here also we can call this as sock because it is not a file descriptor but it is actually socket descriptor.

Reply:
same as above, no diff between socket and file descriptor.

Reply:
why no difference. The underlying mechanism of socket vs file descriptor are different right. Eager to what is the problem if we call it as sd instead of fd?  It is more readable that way.

Reply:
@suresh-thelkar signature of this func is changed as per your comment, so no there is not more fd or sock, means this comment is invalid now.